docs/walkthrough/index.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+title: Walkthroughs & Tutorials
+---
+
+## Create a Resource Group
+
+Install the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) if you don't already have it.
+
+```bash
+az group create --name PromitorRG --location eastus
+```
+
+Output: 
+```json
+{ 
+  "id": "/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG",
+  "...": "..."
+}
+```
+
+## Create a Service Principal
+
+Use the resource group creation output to add a scope to your service principal:
+
+```bash
+az ad sp create-for-rbac \
+  --role="Monitoring Reader" \
+  --scopes="/subscriptions/<guid-subscription-id>/resourceGroups/PromitorRG"
+```
+
+Which should output something similar to
+
+```json
+{
+  "appId": "<guid-sp-app-id>",
+  "displayName": "azure-cli-2019-03-29-19-21-58",
+  "name": "http://azure-cli-2019-03-29-19-21-58",
+  "password": "<guid-sp-generated-password>",
+  "tenant": "<guid-tenant-id>"
+}
+```
+
+Save this output - the app ID, tenant ID, and password will be used later.
+
+## Create a Service Bus Namespace and Queue
+
+(From the [Service Bus Quickstart](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-cli))
```

Comment:
I think this is very interesting but less relevant for this walkthrough.

We can either add a `Further Reading` section at the end or just remove it I think.